commit f0aa78ca1b2b32478181468c5fe6b3d2fa1cf676
Author: Theodore Ts'o <tytso@mit.edu>
Date:   Sat Feb 4 23:38:06 2017 -0500

    ext4: preserve the needs_recovery flag when the journal is aborted
    
    commit 97abd7d4b5d9c48ec15c425485f054e1c15e591b upstream.
    
    If the journal is aborted, the needs_recovery feature flag should not
    be removed.  Otherwise, it's the journal might not get replayed and
    this could lead to more data getting lost.
    
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1282_f0aa78ca1b2b32478181468c5fe6b3d2fa1cf676.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1282_f0aa78ca1b2b32478181468c5fe6b3d2fa1cf676.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
